docs/flavors.md

```diff
@@ -2,50 +2,52 @@
 
 ## Android
 
-To add a new flavor, you need to open [`android/app/build.gradle`](android/app/build.gradle) file and find the following code:
+To add a new flavor, you need to open [`android/app/build.gradle`](android/app/build.gradle) file
+and find the following code:
 
 ```groovy   
  flavorDimensions "release-type"
 
- productFlavors {
-     dev {
-         dimension "release-type"
-         applicationIdSuffix ".dev"
-         versionNameSuffix "-dev"
-     }
-     prod {
-         dimension "release-type"
-     }
- }
+productFlavors {
+    dev {
+        dimension "release-type"
+        applicationIdSuffix ".dev"
+        versionNameSuffix "-dev"
+    }
+    prod {
+        dimension "release-type"
+    }
+}
 ```
 
-You have to add your custom flavor to the `productFlavors` section. 
+You have to add your custom flavor to the `productFlavors` section.
 
 For example:
 
 ```groovy
  flavorDimensions "release-type"
 
- productFlavors {
-     dev {
-         dimension "release-type"
-         applicationIdSuffix ".dev"
-         versionNameSuffix "-dev"
-     }
-     prod {
-         dimension "release-type"
-     }
-     qa {
-         dimension "release-type"
-         applicationIdSuffix ".qa"
-         versionNameSuffix "-qa"
-     }
- }
+productFlavors {
+    dev {
+        dimension "release-type"
+        applicationIdSuffix ".dev"
+        versionNameSuffix "-dev"
+    }
+    prod {
+        dimension "release-type"
+    }
+    qa {
+        dimension "release-type"
+        applicationIdSuffix ".qa"
+        versionNameSuffix "-qa"
+    }
```

Comment:
We do not have `qa` flavor anymore in our template anymore. 
https://github.com/surfstudio/surf-flutter-app-template/pull/100